packages/generator/src/generator-options.ts

```diff
@@ -176,6 +195,7 @@ export const generatorOptionsCli: KeysToOptions = {
     describe:
       'By default, the generator will exit when encountering a file that already exists. When set to true, it will be overwritten instead. Please note that compared to the --clearOutputDir option, this will not delete outdated files.',
     type: 'boolean',
+    deprecated: 'Since now. Use overwrite instead.',
```

Comment:
[q] Should we put the next version like 2.11.1 ior so?

Reply:
[q] Should there be a `replacedBy` also for this one?

Reply:
In general it seems that a lot of deprecation changes are missing.

Reply:
Sorry, this was just for a manual test and was not supposed to be here. There is another PR for the deprecation [here](#3137).